research/src/pages/contribute.mdx

```diff
@@ -198,7 +198,7 @@ The `pathToProposal` key tells your research page where its proposal page is hos
 
 Example:
 
-```
+```markdown
```

Comment:
Currently Prism syntax highlighting doesn't do much with this sample, but the next version of Prism will have first class support for highlighting markdown front matter yaml syntax (PrismJS/prism#2634)